app/src/main/kotlin/com/simplemobiletools/flashlight/helpers/MyCameraImpl.kt

```diff
@@ -272,12 +272,12 @@ class MyCameraImpl private constructor(private val context: Context, private var
         handleCameraSetup()
         while (!shouldStroboscopeStop) {
             try {
-                cameraFlash.runOrToast {
+                cameraFlash!!.run {
                     toggleFlashlight(true)
                 }
                 val onDuration = if (isStroboSOS) SOS[sosIndex++ % SOS.size] else stroboFrequency
                 Thread.sleep(onDuration)
-                cameraFlash.runOrToast {
+                cameraFlash!!.run {
```

Comment:
might wanna avoid the !! and use ?

Reply:
Actually wanted it to act as any other error in this case, since that was partly at fault for leaving flashlight in the wrong state. It would take much more refactoring to do it properly unfortunately :/

Reply:
Ah okay